packages/javascript/bh-shared-ui/src/components/PasswordDialog/PasswordDialog.tsx

```diff
@@ -25,11 +25,10 @@ import {
     Grid,
     TextField,
 } from '@mui/material';
-import DialogActions from '@mui/material/DialogActions';
-import DialogContent from '@mui/material/DialogContent';
+import { DialogActions, DialogContent } from '@mui/material';
 import React from 'react';
 import { Controller, useForm } from 'react-hook-form';
-import { PASSWD_REQS, passwordRegex } from 'src/passwdUtils';
+import { PASSWD_REQS, passwordRegex } from '../../utils';
```

Comment:
Relative import here

Reply:
I took a look into this and now that this component lives in `bh-shared-ui` changing it to use an absolute import requires a significant number of changes to configuration and how the libraries and the main ui are built.

While I agree it is better practice to use absolute imports, the amount of work involved to accommodate that change should probably come in its own PR